assets/scss/common/_campaign-thumbnail-block.scss

```diff
@@ -78,6 +78,10 @@
         width: 100%;
         max-height: 260px;
         height: auto;
+
+        @media (max-width: #{$medium-width}) {
+          margin: -50px 0 -10px 0;
```

Comment:
Why not use our responsive mixins? You can add these margins to `img` and use `@include medium-and-up` to reset them.

Reply:
@comzeradd Thanks for the solution.